trivy-task/trivyV1/task.json

```diff
@@ -129,9 +131,9 @@
       "label": "Scan Type",
       "required": false,
       "options": {
-        "image": "Image",
```

Comment:
probably, it's better to not change this values in v1

Reply:
yup - good call